logging/google/cloud/logging/entries.py

```diff
@@ -74,7 +77,13 @@ class _BaseEntry(object):
     """
     def __init__(self, payload, logger, insert_id=None, timestamp=None,
                  labels=None, severity=None, http_request=None):
-        self.payload = payload
+        if isinstance(payload, any_pb2.Any):
```

Comment:
@lukesneeringer @tseaver I am "sad" about this change in that it incurs an `isinstance` penalty for every single user of this base class.